### PR TITLE
DIV-6509-Update name and descriptions

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6509

### Change description ###
From Jeremy
coRespAnswerReceivedFromServiceAppRejected' label should be the same string as the name 
'Answer received - Co-Respondent'. 

_**However, this creates a 31 character limit error, so i have shorted it to 
'Answer received - CoRespondent'.**_ 

    IssueAOSOffline_Respondent_FromServiceAppRejected:
+ should be **"Description": "Issue AOS pack to respondent"** 
- not "Description": "Answer received"
    
IssueAOSOffline_CoRespondent_FromServiceAppRejected
+ should be **"Description": "Issue AOS pack - Co-respondent"** 
- not "Description": "Answer received"

### CODE REVIEW

**Review by madjava — APPROVED** [review]

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json [hunk 1]
```diff
@@ -201,8 +201,8 @@
     "LiveFrom": "16/09/2020",
     "CaseTypeID": "DIVORCE",
     "ID": "coRespAnswerReceivedFromServiceAppRejected",
-    "Name": "Answer received",
-    "Description": "Answer received",
+    "Name": "Answer received - CoRespondent",
+    "Description": "Answer received - CoRespondent",
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:205]: Description should be: "Answer received from Co-Respondent"

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json [hunk 2]
```diff
@@ -213,7 +213,7 @@
     "CaseTypeID": "DIVORCE",
     "ID": "IssueAOSOffline_Respondent_FromServiceAppRejected",
     "Name": "Issue AOS pack to respondent",
-    "Description": "Answer received",
+    "Description": "Issue AOS pack to respondent",
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:216]: Here should be "Issue AOS (offline) for respondent from ServiceApplicationNotApproved" to follow convention

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json [hunk 3]
```diff
@@ -225,7 +225,7 @@
     "CaseTypeID": "DIVORCE",
     "ID": "IssueAOSOffline_CoRespondent_FromServiceAppRejected",
     "Name": "Issue AOS pack - Co-respondent",
-    "Description": "Answer received",
+    "Description": "Issue AOS pack - Co-respondent",
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:228]: Here, should be: "Issue AOS (offline) for Co-Respondent from ServiceApplicationNotApproved"
Unless Jeremy requested to change it that way :)

**Review by Dizrupto — CHANGES_REQUESTED** [review]

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json (hunk 1 above)
Comment (jacmurphy) [reply definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:205]: Please read the description of the PR. I also commented on the ticket

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json (hunk 3 above)
Comment (jacmurphy) [reply definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:228]: The ACs state,
"Name": "Issue AOS pack - Co-respondent", 
"Description": "Issue AOS pack - Co-respondent",

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json (hunk 2 above)
Comment (jacmurphy) [reply definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:216]: The ACs state:
"Name": "Issue AOS pack to respondent", 
"Description": "Issue AOS pack to respondent",

File: definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json (hunk 1 above)
Comment (Dizrupto) [reply definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json:205]: Oh okay nice, yep that's okay 👌

**Review by Dizrupto — APPROVED** [review]

**Pull request merged**